miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -209,6 +209,13 @@
   jQuery(document).ready(function () {
     // Attach Parsley form validator
     Validate.attachParsley('#pool-form');
+
+    // store duplicate indices sequences
+    Pool.duplicateIndices = ${pool.hasDuplicateIndices()};
```

Comment:
Should this kind of data go into the `config` object passed to the `ListUtils.createTable` function rather than the global `Pool` object?